packages/eslint-plugin/src/configs/react.js

```diff
@@ -264,6 +282,8 @@ const config = {
     'import/named': 'off',
     'import/namespace': 'off',
     'import/no-named-as-default-member': 'off',
+
+    'jsdoc/check-tag-names': ['error', { definedTags: ['jest-environment'] }],
```

Comment:
I'd remove that definedTags and instead turn off that rule in particular file - this is rather an edge case - a good practice is not to generalise on edge cases from my experience